.gitignore

```diff
@@ -45,6 +45,7 @@ jspm_packages
 # Misc
 .DS_Store
 package-lock.json
+nodemon.json
```

Comment:
Was this change intentional?

Reply:
Yes. I have nodemon configs locally and don't want to push them up.